pravega/pravega
# Issue 7011: Pravega LTS based Recovery

**Change log description**  
PR to recover Pravega from LTS.
More on #7011 

**Purpose of the change**  
Fixes #7011 

**What the code does**  
Introduces a CLI command using which one can "recover Pravega" from an existing Tier-2 mount.  Document in this PR (documentation/src/docs/recovery-procedures/lts-recovery-steps.md) describes the procedure.

**How to verify it**  
Command should successfully run.
An example of a sample run showing events written to a stream before recovery and read from the same recovered stream after recovery( ommiting steps that install/uninstall Pravega)

Before Recovery:
Creating a sample stream and adding some events.
![image](https://user-images.githubusercontent.com/88840194/225536280-a55841c3-36e5-404c-a99e-d1e0563ec53f.png)

Running Recovery:
![image](https://user-images.githubusercontent.com/88840194/225537283-55d16733-c17a-48b5-9eb2-12f920ba7a5a.png)
Sample Output for a container:
![image](https://user-images.githubusercontent.com/88840194/225537425-9c0640fe-3ada-4c75-9ec0-fa3ab5a1ddc3.png)


After Recovery:
Reading the recovered stream.
![image](https://user-images.githubusercontent.com/88840194/225537592-6cae117f-d5ac-4c23-83c7-786ad0dc8687.png)
